app/master/time_based_atom_grouper.py

```diff
@@ -103,28 +103,27 @@ def groupings(self):
     def _coalesce_new_atoms_with_historic_times(self, new_atoms, old_atoms_with_times, project_directory):
```

Comment:
I remember naming this method. I think it wasn't a great name then, and it's an even worse name now with this change.

Maybe we can rename it to _set_expected_times()?